feature/auth/build.gradle.kts

```diff
@@ -43,5 +56,17 @@ kotlin {
 
             implementation(libs.play.services.auth)
         }
+
+        commonTest.dependencies {
+            implementation(kotlin("test-common"))
+            implementation(kotlin("test-annotations-common"))
+            implementation(libs.kotlinx.coroutines.test)
```

Comment:
- Add the dependencies if it is needs for example `implementation(libs.kotlinx.coroutines.test)` this one is already added in `KMPLibraryConventionPlugin.kt` or `implementation(compose.uiTest)` as we are not focusing in UI test don't add it and other dependecnies that you added.
- @niyajali should he use `mokerry` third party library for testing?